<launch>

    <node name="aruco_tf2_1" pkg="aruco_odom" type="aruco_tf2.py" respawn="false" output="screen" >
        <param name="map_file" type = "string" value="/home/rock/catkin_ws/src/aruco_odom/nodes/C4S_map.txt"/>
        <param name="fx_1" type = "string" value="285.222808837891"/>
        <param name="fy_1" type = "string" value="285.393310546875"/>
        <param name="ppx_1" type = "string" value="424.528411865234"/>
        <param name="ppy_1" type = "string" value="405.992706298828"/>
        <param name="c1_1" type = "string" value="-0.0040830047801137"/>
        <param name="c2_1" type = "string" value="0.0394832715392113"/>
        <param name="c3_1" type = "string" value="-0.0364591591060162"/>
        <param name="c4_1" type = "string" value="0.00567311979830265"/>
        <param name="fx_2" type = "string" value="285.219696044922"/>
        <param name="fy_2" type = "string" value="285.371307373047"/>
        <param name="ppx_2" type = "string" value="426.107391357422"/>
        <param name="ppy_2" type = "string" value="402.454803466797"/>
        <param name="c1_2" type = "string" value="-0.0029170720372349"/>
        <param name="c2_2" type = "string" value="0.0377450212836266"/>
        <param name="c3_2" type = "string" value="-0.0351304709911346"/>
        <param name="c4_2" type = "string" value="0.0052577848546207"/>
    </node>
</launch>  